controllers/remove_process_groups_test.go

```diff
@@ -489,17 +501,70 @@ var _ = Describe("remove_process_groups", func() {
 					Expect(removedProcessGroup.ProcessGroupID).To(Equal(fdbv1beta2.ProcessGroupID("storage-1")))
 					removedProcessGroup.MarkForRemoval()
 					cluster.Status.ProcessGroups[0] = removedProcessGroup
+					status.Cluster.DatabaseConfiguration.ExcludedServers = append(status.Cluster.DatabaseConfiguration.ExcludedServers, fdbv1beta2.ExcludedServers{Locality: removedProcessGroup.GetExclusionString()})
```

Comment:
Why are we doing an append here? Do we expect that the `ExcludedServers` contain any elements?